en/tools/config-catalog-generator/package-lock.json

```diff
@@ -1,8 +1,403 @@
 {
   "name": "gen-config-page",
   "version": "5.8.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Is it correct that this file has been added?

Reply:
This file is required to generate the config catalog. Contains updated info about the node modules.